src/Runner/PhptTestCase.php

```diff
@@ -66,7 +66,7 @@ class PhptTestCase implements Test, SelfDescribing
      * Constructs a test case with the given filename.
      *
      * @param string             $filename
-     * @param AbstractPhpProcess $phpUtil
+     * @param \PHPUnit\Util\PHP  $phpUtil
```

Comment:
changing param type for regular method is BC breaker.
@sebastianbergmann , do you treat changing `__construct` params as BC breaker ? I know some do, some don;t

Reply:
This is a good question!